prisma/schema.prisma

```diff
@@ -30,6 +30,7 @@ model EventType {
   teamId                  Int?
   bookings                Booking[]
   availability            Availability[]
+  webhooks                WebhookEventTypes[]
```

Comment:
Do we need to add webhooks to the EventType model? I'm thinking it may make sense to keep it entirely separate; If the question is "Does the event type ever need to interact directly with webhooks?

Maybe in the future we want to have webhooks that have nothing to do with the EventType;  For example when an account hits a sync error, or a added calendar connection.

Reply:
I think that if we keep the webhooks restricted to following eventType based hooks only, then directly associating them with the eventType makes sense. And that was the case based on early interactions I had with everyone. However, what you say makes sense that if we want to create hooks for "sync error" or something else, then we'd have to reformulate this a bit.

Reply:
You're right; it's OK for now; But lets make sure we add all linking data in the event payload too so we can one day remove the eventTypes link if we so wish.